### ISSUE TITLE
NotImplementedException when using @RequestScope()

### ISSUE THREAD

**Issue by AltarBeastiful** [kugacz/nj-request-scope#4]
I'm getting compilation error when using @RequestScope() decorator.
Is there a missing part I should implement to use this ? 

```
node_modules/nj-request-scope/lib/util/dynamic.object.handler.factory.js:27
            throw new NotImplementedException();
                  ^
Error: Not implemented. Feel free to implement.
    at Object.getPrototypeOf (//node_modules/nj-request-scope/lib/util/dynamic.object.handler.factory.js:27:19)
    at Function.[Symbol.hasInstance] (<anonymous>)
    at mapToProxy (node_modules/@nestjs/core/nest-factory.js:167:35)
    at Proxy.<anonymous> (//node_modules/@nestjs/core/nest-factory.js:182:32)
    at bootstrap (/src/main.ts:39:16)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

```

**Comment by kugacz** [comment]
Hi @AltarBeastiful,
Initially, I implemented only those Proxy methods I needed. The rest of them are throwing a "Not implemented. Feel free to implement" error.
I've prepared a new version of nj-request-scope with 'getPrototypeOf' implementation, which you need.
Please, try the new release candidate version:  1.0.4-rc.0 and let me know if all is working properly.

**Comment by AltarBeastiful** [comment]
Indeed it works ! 
Will try to contribute if I'm stuck again on one.

**Comment by kugacz** [comment]
It's great to hear :) Thank you for testing!
I've released a final version with this fix: 1.0.4.
